runtime/gc_base/GCExtensions.hpp

```diff
@@ -307,7 +307,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	 * @param[in] isGlobalGC
 	 * @return true if we need to scan the java stack
 	 */
-	static bool needScanStacksForContinuationObject(J9VMThread *vmThread, j9object_t objectPtr, bool isGlobalGC);
+	static bool needScanStacksForContinuationObject(J9VMThread *vmThread, j9object_t objectPtr, bool isConcurrentGC, bool isGlobalGC, bool beingMounted = false);
```

Comment:
Can we remove default defined value for parameter and make all calls be 5 parameters explicitly?
It will help to avoid misinterpretation in the case one of bool parameters is missed